sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -18,6 +18,21 @@ export interface StudentFinancialInformationAppealData extends JSONDoc {
   daycareCosts11YearsOrUnder?: number;
 }
 
+export enum TransportationCostSituation {
+  NotRequired = "notRequired",
```

Comment:
Same here.

Reply:
I can still see the enum item, it is supposed to be removed already?

Reply:
I'm sorry. Removed.